src/test/java/com/gizasystems/automationexercise/tests/SearchProducts/SearchProductsandVerifyCartAfterLogin.java

```diff
@@ -0,0 +1,78 @@
+package com.gizasystems.automationexercise.tests.SearchProducts;
+
+
+import com.gizasystems.automationexercise.pages.*;
+import com.shaft.driver.SHAFT;
+import io.qameta.allure.Epic;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+@Epic("Automation Exercise")
+@Feature("Products Page")
```

Comment:
There's no feature for a single page, please refactor this to something more descriptive.